src/pages/dashboard/e2e.dashboard.spec.ts

```diff
@@ -51,37 +57,261 @@ describe("Dashboard", () => {
 
   it("Should go to the Debug page", async () => {
     await page.goto("http://localhost:3000");
-    await page.waitForSelector("#header-links");
+    await page.waitForSelector("#header-links-default");
     await page.click("a[href='/debug']");
     const currentUrl = page.url();
     expect(currentUrl).toBe("http://localhost:3000/debug?tileset=san-francisco-v1.7");
     const toolBar = await page.$$("#tool-bar");
     expect(toolBar).toBeDefined();
   });
 
-  it("Should go to the Comparison page", async () => {
+  // TODO turn it on back then compare dropdown menu will be implemented
+
+  // it("Should go to the Comparison page", async () => {
```

Comment:
This test will completely overwritten. I suggest removal this one

Reply:
It will be removed in next [PR](https://github.com/visgl/loaders.gl-showcases/pull/75)